Source/WebCore/platform/graphics/GlyphPage.h

```diff
@@ -119,12 +130,14 @@ class GlyphPage : public RefCounted<GlyphPage> {
 private:
     explicit GlyphPage(const Font& font)
         : m_font(font)
+        , m_isColor(size)
     {
         ++s_count;
     }
 
     const Font& m_font;
     Glyph m_glyphs[size] { };
+    BitVector m_isColor; // FIXME: This is too large for WTF::BitVector to keep the storage inline. Either use std::bitset or adjust WTF::BitVector.
```

Comment:
This should migrate to `WTF::Bitmap`

Reply:
https://github.com/WebKit/WebKit/commit/da4d5fa288fc590dd6d77d598d66a61de11b6c2c